### PR TITLE
Dashboard ui Design

### PR DESCRIPTION
**Pull request by AbhijitP98**

### CODE REVIEW

File: src/components/molecules/DecisionCard/index.jsx [hunk 1]
```diff
@@ -57,6 +57,9 @@ const useStyles = makeStyles({
     clocklogo: {
         paddingRight: '0.2rem',
         marginBottom: '0.1rem'
+    },
+    ulist: {
```
Comment (NaveenReliable) [inline src/components/molecules/DecisionCard/index.jsx:61]: use proper naming here for class name

File: src/components/molecules/DecisionCard/index.jsx [hunk 2]
```diff
@@ -66,13 +69,13 @@ const DecisionCard = ({ data }) => {
...
         <Grid item xs={4} className={classes.decisionGrid}>
-            <Link to={`/decision/${data.id}`}>
+            {/* <Link to={`/decision/${data.id}`}> */}
             <Grid className={classes.cardGrid}>
                 <h2>{data.heading}</h2>
                 <h2 className={classes.subHead}>{data.subHeading}</h2>
                 <p className={classes.description}>Decision impact:</p>
                 <span className={classes.list}>
-                    <ul>
+                    <ul className={classes.ulist}>
```
Comment (NaveenReliable) [inline src/components/molecules/DecisionCard/index.jsx:78]: Follow the same comments which is mentioned above.

**Review by NaveenReliable — CHANGES_REQUESTED** [review]

File: src/components/molecules/DecisionCard/index.jsx (hunk 1 above)
Comment (AbhijitP98) [reply src/components/molecules/DecisionCard/index.jsx]: In latest PR request this is fixed

File: src/components/molecules/DecisionCard/index.jsx (hunk 2 above)
Comment (AbhijitP98) [reply src/components/molecules/DecisionCard/index.jsx]: In latest PR request this is fixed ,so i'm closed this pr

File: src/components/molecules/DecisionCard/index.jsx [hunk 3]
```diff
@@ -87,7 +90,7 @@ const DecisionCard = ({ data }) => {
                     </p>
                 </Grid>
             </Grid>
-            </Link>
+            {/* </Link> */}
```
[the remark this answers is not in the document]
Comment (AbhijitP98) [reply src/components/molecules/DecisionCard/index.jsx]: In latest PR request this is fixed ,so i'm closed this pr

**Pull request merged**